test/engine/engine_io_test.cc

```diff
@@ -47,6 +51,34 @@ mjModel PartialModel(const mjModel* m) {
   return partial_model;
 }
 
+TEST_F(EngineIoTest, VerifySizeModel) {
+  constexpr char xml[] = R"(
+  <mujoco>
+    <worldbody>
+      <body>
+        <joint/>
+        <geom size="1"/>
+      </body>
+    </worldbody>
+  </mujoco>
+  )";
+
+  std::array<char, 1024> error;
+  mjModel* model = LoadModelFromString(xml, error.data(), error.size());
+  ASSERT_THAT(model, NotNull()) << "Failed to load model: " << error.data();
+
+  std::filesystem::path temp_file = (
```

Comment:
This test fails on Windows (with MSVC). We have another test that creates temporary files like this [here](https://github.com/deepmind/mujoco/blob/c17375f/test/fixture.cc#L112).

Maybe you can find out why that utility works and this test doesn't? Maybe you'll need to explicitly convert the `std::filesystem::path` it to a `std::string` by calling `.string()`.

Reply:
You were right, adding `.string()` fixed the issue.